tests/fixtures/explicit-imports/test-app/tests/unit/import-test.ts

```diff
@@ -5,7 +5,7 @@ import { hbs } from 'ember-cli-htmlbars';
 
 import * as myModule from 'my-addon';
 
-module('imports', function (hooks) {
```

Comment:
this is the only fix not associated with the goal of this PR -- hooks is an unused variable